src/main/java/net/minecraftforge/common/util/EnumHelper.java

```diff
@@ -141,6 +143,19 @@ public static EnumRarity addRarity(String name, TextFormatting color, String dis
     {
         return addEnum(EnumRarity.class, name, color, displayName);
     }
+    /**
+     * 
+     * @param name - The name the new enum constant should have
+     * @param textureLocation - The path to the texture for this armor type. Formatted as domain:path.
+     * @param hash - A short unique 3 letter name for the armor type 
```

Comment:
In vanilla, the hash is used for the horse texture prefix. 
Is that still relevant here, or should we hard-code it to `""` instead of making it a parameter here?